core/templates/dev/head/pages/splash.html

```diff
@@ -8,30 +8,114 @@
-      <div class="oppia-gallery-banner-content" style="height: 100%; width: 100%;">
-        Hi, I&rsquo;m Oppia.
-        <br>
-        <span style="font-size: 0.8em;">
-          This splash page needs to be replaced.
-        </span>
-      </div>
+
+  <div id="oppia-splash-section-one" class="text-center">
+    <h1 class="oppia-splash-h1">Think outside the books.</h1>
+    <div class="oppia-splash-background">
+      <img src="images/splash/Economics.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Computing.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Law.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Art.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Creativity.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Physics.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Language.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Arithmetic.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Chess.svg" class="oppia-splash-back-images" />
+
+      <img src="images/splash/Geography.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Biology.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Genetics.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Space.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Algebra.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Music.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Chemistry.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Poetry.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Puzzles.svg" class="oppia-splash-back-images" />
+
+      <img src="images/splash/Humor.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Philosophy.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Mathematics.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Religion.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Cooking.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Engineering.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Microbiology.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/History.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Architecture.svg" class="oppia-splash-back-images" />
+    </div>
+    <img src="images/splash/books.svg" id="oppia-splash-books" />
+    <h2 class="oppia-splash-h2">Oppia makes it easy to create interactive explorations<br>that educate and engage.</h2>
+    <div class="row oppia-splash-buttons-row">
+      <button class="oppia-splash-buttons">BROWSE OUR EXPLORATIONS</button>
```

Comment:
These buttons don't go anywhere. Can you add their destinations?

"Browse" should go to "/gallery".
"Create" should go to "/my_explorations?mode=create".